docker/oms-events/bootstrap.sh

```diff
@@ -0,0 +1,36 @@
+#!/bin/bash
```

Comment:
I'm actually not quite sure if this file is needed, since the commands that are here are in Dockerfile and I think that's the better way than using bootstrapping script for it.

Reply:
Again, did not critically look at this, if what you say is true I fully agree you.

Reply:
@blacksph3re is the bootstrapping file used anywhere? I'm quite sure it isn't, but still.

Reply:
I think we can safely discard it, especially if you agree with my idea of synchronizing startups through health requests which I posted in slack. Then we could also leave out the strapstate stuff

Reply:
@blacksph3re totally agree, yeah, that should make things more clear.